From c74a08dab73c9ee5dbd4a428325bb7045a905032 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@lucaweiss.eu>
Date: Thu, 26 Jun 2025 18:53:00 +0200
Subject: [PATCH] Bump minimum required CMake version

CMake 4.0.3 does not build this project anymore due to "Compatibility
with CMake < 3.5 has been removed from CMake.".

Bump the minimum version to 3.10, picked semi-arbitrarily.
---
 CMakeLists.txt                    | 2 +-
 cmake/modules/UseLttngGenTp.cmake | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 67289888..4e102d6f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.10)
 
 project(qtmir)
 
diff --git a/cmake/modules/UseLttngGenTp.cmake b/cmake/modules/UseLttngGenTp.cmake
index 75ad8f4c..44296141 100644
--- a/cmake/modules/UseLttngGenTp.cmake
+++ b/cmake/modules/UseLttngGenTp.cmake
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 3.10)
 if(POLICY CMP0011)
  cmake_policy(SET CMP0011 NEW)
 endif(POLICY CMP0011)
-- 
2.50.0

